code/modules/events/blob.dm

```diff
@@ -1,10 +1,9 @@
 /datum/round_event_control/blob
 	name = "Blob"
 	typepath = /datum/round_event/blob
-	weight = 5
 	max_occurrences = 1
 
-	earliest_start = 48000 // 1 hour 20 minutes
+	average_time = 80 //High potential to end rounds; only spawn in rounds that have gone on too long.

```

Comment:
Blobs should be roughly the same time as aliens, as they're roughly the same amount of threat, if in different packages.